common/djangoapps/student/auth.py

```diff
@@ -20,7 +20,7 @@
 # In addition to the above, one is always allowed to "demote" oneself to a lower role within a course, or remove oneself
 
 
-def has_access(user, role):
+def has_role(user, role):
```

Comment:
Terminology-wise:
"a user *has* a role"
"a role has permissions"
"a user has access to a resource given the user's role and the role's permissions"

So I would leave this is as "has_role" or to be more explicit, "user_has_role".